src/android/CHIPTool/app/src/main/AndroidManifest.xml

```diff
@@ -22,6 +23,15 @@
             android:name=".OnOffActivity"
             android:exported="false"
             android:screenOrientation="portrait"/>
+
+        <activity
+            android:name="com.google.chip.chiptool.commissioner.CommissionerActivity"
+            android:label="@string/commissioner_name">
+            <intent-filter>
```

Comment:
Now that the CommissionerActivity is not being started via an Intent action, this intent-filter can be removed.

Reply:
removed.